azerothcore/aowow
# Add account support for AzerothCore auth database

PR introduces changes to allow aowow accounts and authenticate with AzerothCore auth/account database tables. The update ensures compatibility with AzerothCore, including user verification and ban checks.

- Updated error handling to provide more detailed logs and error messages.
- Maintained compatibility with the existing TrinityCore setup, ensuring that both database structures are supported without conflict.

To test:
- Configure Aowow to connect to an AzerothCore auth/account database.
- Attempt to log in with a valid user and verify successful authentication.
- Check the behavior with a banned user to ensure proper handling and error messages.
- Repeat the above steps with a TrinityCore database to confirm backward compatibility.

Feel free to request changes or edit.
